src/routes/fassungen/[thirties]/FassungskommentarModal.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+	import { Modal } from '@skeletonlabs/skeleton-svelte';
+
+	let { commentary, id, openState = $bindable() } = $props();
+</script>
+
+<Modal
+	defaultOpen={true}
+	onOpenChange={() => (openState = false)}
+	classes="fassungskommentar_modal"
+	triggerBase="btn preset-tonal"
+	contentBase="w-[80vw] card bg-surface-100-900 p-4 space-y-4 shadow-xl max-w-screen-sm"
+	base=""
+	backdropClasses="backdrop-blur-sm"
+>
+	{#snippet content()}
+		{#if id[2] !== 'A'}
+			<header class="flex justify-between bg-green-400">
+				<h1 class="h5">Editorischer Kommentar</h1>
+			</header>
+		{/if}
+		<article class="fk-modal-content h-[80vh] overflow-auto">
```

Comment:
```suggestion
		<article class="fk-modal-content max-h-[80vh] overflow-auto">
```

much better then having a lot of empty space

Reply:
@vvvyyynet did you forget to do that or decide not to do that?